packages/pwa-kit-extension-sdk/src/shared/utils/extensibility.ts

```diff
@@ -108,38 +110,11 @@ export const getExtensionNames = (extensions: ApplicationExtensionEntry[]) => {
     })
 }
 
-/**
- * Retrieves information about application extensions from the provided application configuration or,
- * if none is provided, from the default configuration in the project's `package.json`.
- *
- * This function returns an object containing two lists: `installed`, which includes extensions
- * found in the project's dependencies, and `configured`, which lists extensions configured
- * in the application settings.
- *
- * @param {object} [appConfig] - Optional application configuration object. If not provided,
- * defaults to the configuration in the project's `package.json` file under the `mobify` key.
- * @returns {Object} An object containing the following properties:
- * - `installed` {string[]} - An array of installed extension names based on the project's devDependencies.
- * - `configured` {string[]} - An array of configured extension names as specified in the application configuration.
- */
-export const getApplicationExtensionInfo = (appConfig?: any) => {
+export const getPackageNamesOfInstalledExtensions = () => {
```

Comment:
NIT: This name is is quite verbose. Correct me if I'm wrong but we an extension name is synonymous with the package name.. I think. This is because we stepped back from having that "short" name concept. Would calling this `getInstalledExtensions` and returning the expanded version be more useful. If you need jus the name in a function that is consuming this you can easily parse them out of the tuples? Just a thought.

Reply:
Also we has a bunch of nice looking readable block comments.. it would be nice to keep them.